src/expressions.rst

```diff
@@ -4424,19 +4438,11 @@ A :t:`return expression` is an :t:`expression` that optionally yields a
 :dp:`fls_5v3j5ghhw8j8`
 A :t:`return expression` shall appear within a :t:`control flow boundary`.
 
-:dp:`fls_m4e00bju2dy4`
-The :t:`type` of a :t:`return expression` is determined as follows:
-
-* :dp:`fls_xpp027s2m7ue`
-  If the :t:`return expression` has an :t:`operand`, then the :t:`type` is the
-  :t:`type` of the :t:`operand`.
-
-* :dp:`fls_cqduumpsjfut`
-  If the :t:`return expression` does not have an :t:`operand`, then the
-  :t:`type` is the :t:`never type`.
+:dp:`fls_7Ck4LMQMeQCv`
+The :t:`type` of a :t:`return expression` is the :t:`never type`.
 
 :dp:`fls_r610t5vsi7bx`
-The :t:`value` of a :t:`return expression` is determined as follows:
+The returned :t:`value` of a :t:`return expression` is determined as follows:
```

Comment:
It might be better to reword this as

The value returned by a return expression is determined as follows: